src/visualization/manipulator.cc

```diff
@@ -288,7 +290,6 @@ bool FreeManipulator::mouseEvent(QObject* watched, QMouseEvent* event) {
         }
         prev_pos_ = event->pos();
       }
-      return true;

```

Comment:
Why those changes to don't filter out those handled mouse events (we still filter out keyboard/wheel though)?